rpcs3/Emu/Cell/lv2/sys_storage.cpp

```diff
@@ -46,13 +91,22 @@ error_code sys_storage_async_configure(u32 fd, u32 io_buf, u32 equeue_id, u32 un
 {
 	sys_storage.todo("sys_storage_async_configure(fd=0x%x, io_buf=0x%x, equeue_id=0x%x, unk=*0x%x)", fd, io_buf, equeue_id, unk);
 
+	auto& manager = g_fxo->get<storage_manager>();
+	manager.asyncequeue = equeue_id;
```

Comment:
Use the usual idm::get<lv2_obj, lv2_event_queue>(equeue_id) with ESRCH should suffuc, the type of storage_manager.asyncqueue should be atomic_ptr<std::shared_ptr<lv2_event_queue>>.